### ISSUE TITLE
A petition request is submitted to Shenzhen Human Resources and Social Security Bureau

### ISSUE THREAD

**Issue by scottwansz** [xokctah/996.petition#9]
It is submitted as public, but not sure if it can be accessed publicly, the request# is 20190413-3797-16779775
The website submit this is http://www.sz.gov.cn/cn/
by 'Web Petition'.

**Comment by xokctah** [comment]
Nope. Can you copy&paste and/or screenshot the request, so that we can make
it public?

**Comment by scottwansz** [comment]
Screenshot of the request
--------------------------
https://github.com/anti996sz/996.ICU/blob/master/blacklist/petiton_to_sz_hr_beau_nbr.png
https://github.com/anti996sz/996.ICU/blob/master/blacklist/petiton_to_sz_hr_beau.png

**Comment by scottwansz** [comment]
我建议其它城市的朋友，也像我一样，通过正当的渠道提交投述请求，这样也许也能形成一股立量。

**Comment by xokctah** [comment]
赞！非常好的行动！

我有个想法，我们能不能把各个城市的投诉渠道列出来，写一个简单的投诉模板，让大家可以很简单的点开链接，copy paste就可以投诉。

有时间的同学可以提PR，我最近有点忙……

**Comment by scottwansz** [comment]
https://github.com/anti996sz/996.ICU/blob/master/blacklist/anser1.jpg
https://github.com/anti996sz/996.ICU/blob/master/blacklist/answer2.jpg

**Comment by scottwansz** [comment]
处理状态 	已办结
咨询内容 	请核实并处理深圳违法用工情况，具体违法企业及举报证据可见： https://github.com/996icu/996.ICU/tree/master/blacklist
回复内容 	

你好，根据市区事权划分有关规定和属地管理原则，如用人单位违反劳动法有关规定的，请直接向当地人力资源保障部门反映，或通过市政府信箱https://szum.sz.gov.cn:9443/personalCenter/#/login或http://sso.sz.gov.cn/pub/szol/hall/）反映，由市政府信访部门转由相关部门处理。

附：各部门联系电话及地址：

市社保局            电话：12333     地址：福田区深南大道8005号深圳人才园二楼信访大厅，信访邮箱：csixf@hrss.sz.gov.cn。

福田区人力资源局 地址：深圳市福田区福民路123号区委东大门　电话：82918158，82918291-207（劳动信访调解）

罗湖区人力资源局  地址：深圳市罗湖区沿河北路2019 号（原罗湖区人民法院） 电话： 25438399

盐田区人力资源局  地址：深圳市盐田区深盐路区信访大厅（政府广场东南侧） 电话：25228211（劳动信访）；25228431（劳动监察投诉举报）

盐田区委组织公务员与职业管理办公室 地址：深圳市深盐路2088号区行政中心714房 电话：25228714

南山区人力资源局  地址：深圳市南山区深南大道12017号劳动大厦一楼东侧 电话：26665374

宝安区人力资源局  地址：深圳市宝安区新安街道6区前进路91号宝安区公共就业管理中心2楼信访窗口　电话： 29990000　

龙岗区人力资源局  地址：深圳市龙岗区中心城龙翔大道8027号  电话：28917170

光明新区统战和社会建设局  地址：深圳市光明新区碧眼1号信访大厅　电话：88211835

光明新区组织人事局  地址：深圳市塘明路1号光明新区管委会第二办公区3栋207  电话：88211369

坪山区社会建设局  地址：深圳市坪山区金牛西路华瀚大厦1楼信访大厅  电话：84622204

龙华区人力资源局  地址：深圳市龙华区龙华街道清龙路8号劳动诉求服务中心  电话：23336155（劳动监察）

龙华区组织人事局  地址：深圳市龙华区观澜美术馆一楼　电话：23332222

大鹏新区组织人事局  地址：深圳市大鹏新区葵涌街道金岭路1号大鹏新区管委会5213室电话：28333131

大鹏新区社会建设局  地址：深圳市大鹏新区葵涌街道金岭路1号（大鹏新区管委会1217A）电话：28333104

如有疑问，请拨打人力资源保障服务热线12333咨询。
回复单位 	深圳市人力资源和社会保障局
回复时间 	2019.04.15
